### ISSUE TITLE
Droid code error in compiling

### ISSUE THREAD

**Issue by benschechtman** [XRobots/BB83_Public#2]
hello, im a big fan of your youtube channel, but i have recently started robotics projects. i was trying to use you droid code (droid013) but there is always an error compiling. i have personally checked the entire code, but im no expert.any suggestions? thanks in adavance

**Comment by XRobots** [comment]
Hi,

What's the error - have you got all the libraries installed etc?

-James

**Comment by benschechtman** [comment]
Yes, I have all the libraries installed, but the I just get an error
compiling notification at the end, I'm not sure why

**Comment by XRobots** [comment]
There should be more to the error, obviously it all works fine for me so I
need some more information.

**Comment by benschechtman** [comment]
I'll make sure to look into it more in the morning, it's more than likely
something I messed up when installing the libraries. Thanks again.
